HACKING.md

```diff
@@ -352,6 +352,18 @@ Insert this text before the `</dict>`:
 ```
 
 
+## Building a Generic Goldstone VM
+
+* work in progress *
+
+Follow instructions in the [INSTALL](http://goldstone-server.readthedocs.org/en/latest/INSTALL/) guide for installing via an RPM inside your VM; however, instead of executing `fab install`, execute:
+
+    root# cd /opt/goldstone
+    root# . bin/activate
+    root# fab partial_install::django_admin_password=YOUR_PASSWORD
```

Comment:
There's only one colon after the "partial_install". Should be:

    root# fab partial_install:django_admin_password=YOUR_PASSWORD